src/main/java/com/demo/MybatisApplication/model/StudentEntity.java

```diff
@@ -0,0 +1,58 @@
+package com.demo.MybatisApplication.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import java.util.ArrayList;
+import java.util.List;
+
+@Data
+//@AllArgsConstructor
+//@NoArgsConstructor
+public class StudentEntity {
+    private Long id;
+    private String studentName;
```

Comment:
```suggestion
    private String name;
```
Since this property is of a student and is present inside the student there is no need to add a student suffix.

Reply:
Done and changed column name